egcg_core/rest_communication.py

```diff
@@ -267,7 +267,7 @@ def close(self):
     def __del__(self):
         try:
             self.close()
-        except ReferenceError:
+        except Exception:
             pass
```

Comment:
@mwhamgenomics Isn't this a little too vast? Maybe we should have a specific Exception pass only?

Reply:
We could try to get to know all the different exception that can be raised in `close` but that would be too much work. 
Within `__del__` all exceptions are ignored anyway and only raise a warning. So the only thing we're doing is silencing the warnings.